src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -49,7 +49,7 @@ public static function register($mode = false)
             'legacy' => array(),
             'other' => array(),
         );
-        $deprecationHandler = function ($type, $msg, $file, $line, $context) use (&$deprecations, $getMode) {
+        $deprecationHandler = function ($type, $msg, $file = null, $line = null, $context = array()) use (&$deprecations, $getMode) {
```

Comment:
This is what the [PHP documentation](http://php.net/manual/en/function.set-error-handler.php) says:

> The third parameter is optional, errfile, which contains the filename that the error was raised in, as a string.
>
> [...]
>
> The fourth parameter is optional, errline, which contains the line number the error was raised at, as an integer.